src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java

```diff
@@ -40,6 +40,8 @@ public static DHEnchantment getEnchantmentInformation() {
         return new DHEnchantment(
             new NamespacedKey(DHCore.getPlugin(DHCore.class), "poison"),
             "Poison",
+            "Axes, Swords",
+            "Poison your enemy on hit. (HEALS UNDEAD)",
```

Comment:
Possible: add MOBS after the "HEALS UNDEAD", since it might be slightly unclear, but this is so subtle it doesn't really matter